packages/vanilla/src/components/card/decisionCard.stories.js

```diff
@@ -67,7 +73,7 @@ const DefaultDecisionCardTemplate = ({ title, actionsLayout, danger }) => {
       <div class="cbp-card__content">
         <div class="cbp-card__header">
           <h4 class="cbp-card__title"><i class="fas fa-circle cbp-margin-right-2x"></i>${title}</h4>
```

Comment:
This just needs the `id` of the `aria-describedby` attribute from the related button

Reply:
added